build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+read -p "Build dataplicity agent from PyPi? " -n 1 -r
+echo
+if [[ ! $REPLY =~ ^[Yy]$ ]]
```

Comment:
Yeah.  It's kind of a classic scripting fubar.  If $REPLY is empty, it's truly empty.  It's not none, and so you'll get weirdness in a script - syntax errors or worse.